src/state/component-repository/components/selectors.js

```diff
@@ -59,12 +59,14 @@ export const getECRComponentLastInstallStatus = (state, props) => {
   return get(state, `componentRepositoryComponents.list[${listIdx}].lastInstallStatus`, '');
 };
 
-export const getECRComponentInstallationStatus = (state, props) => (
-  get(state, `componentRepositoryComponents.installation[${props.component.code}]`, '')
+export const getECRComponentInstallationStatus = createSelector(
+  [state => state.componentRepositoryComponents.installation, (state, props) => props.component],
+  (installation, component) => installation[component.code],
 );
 
-export const getECRComponentUninstallStatus = (state, props) => (
-  get(state, `componentRepositoryComponents.uninstallation[${props.component.code}]`, '')
+export const getECRComponentUninstallStatus = createSelector(
+  [state => state.componentRepositoryComponents.uninstallation, (state, props) => props.component],
```

Comment:
looks like `(state, props) => props.component]` is recurring on two selectors. would it be possible to put it in a separate function?

Reply:
Personally I prefer as it is, since its just getting single property from the props. Its more explicit like this since we just have a very simple access..